cookiecutter.json

```diff
@@ -1,11 +1,12 @@
 {
-    "project_name": "project_name",
-    "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
-    "repo_owner": "",
-    "open_source_license": "Not open source",
-    "author_name": "",
-    "description": "My Project Description",
-    "base_docker_image": "manifoldai/orbyter-ml-dev:3.2",
-    "mlflow_uri": "/mnt/experiments",
-    "mlflow_artifact": ""
+  "project_name": "project_name",
+  "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
+  "module_name": "{{ cookiecutter.repo_name.lower().replace(' ', '_').replace('-', '_') }}",
```

Comment:
This is an unfortunate diff. Only `module_name` is added.

Reply:
Or it's because of indenting

Reply:
Shouldn't we just merge the functionality of `module_name` and `repo_name` as per @resdntalien's comment on #37 ?

Reply:
Yes, I would vote to have only `module_name` and have that be both repo_name and module_name.

Reply:
Combined to package_name